src/blob/file.rs

```diff
@@ -69,6 +69,44 @@ impl File {
         self.write_at_aio(offset, buf, ioring).await
     }
 
+    pub(crate) async fn write_append_buffers(&self, buffers: Vec<&[u8]>) -> IOResult<usize> {
```

Comment:
`Vec<>` implies memory allocation. It is better to pass slice of slices. Another possibility: we know that we only need a case for 2 buffers, so we can pass 2 buffers in separate arguments.